applications/DEMApplication/python_scripts/sp_2d_rigid_fem_algorithm.py

```diff
@@ -40,6 +40,15 @@ def Initialize(self):
         with open('sp_porosity.txt', 'w') as poro_file:
             poro_file.write(porosity_message)
 
+        from KratosMultiphysics.DEMApplication.multiaxial_control_module_generalized_2d_utility import MultiaxialControlModuleGeneralized2DUtility
+        self.multiaxial_control_module = MultiaxialControlModuleGeneralized2DUtility(self.spheres_model_part, self.rigid_face_model_part, self.sp_project_parameters)
+        self.multiaxial_control_module.ExecuteInitialize()
+
+    def InitializeSolutionStep(self):
+        super(DEMAnalysisStage2DSpRigidFem, self).InitializeSolutionStep()
+
+        self.multiaxial_control_module.ExecuteInitializeSolutionStep()
```

Comment:
Why do you hardcode this? To me it sees like it would be very suitable to be added as a Process

Reply:
No, it can't be a process because this utility handles various model parts and submodelparts at the same time

Reply:
the Process gets the `Model` in the Constructor, which should contain all the ModelParts used in a simulation

Reply:
Ok, that's true... However, we are using the utility both in the DEMApplication and in the DemStructuresCouplingApplication. And in the latter case we still don't have a DEM-FEM coupling analysis stage neither a DEM-FEM coupling parameters.json, and thus using a process that handles two model parts of two different coupled applications is not straightforward at all...